classes/01class/exercises/c01-aws02/anwickes/README.md

```diff
@@ -0,0 +1,104 @@
+## c01-aws02
+
+- Use the CLI to create a S3 bucket and upload a file to it.
+- Access the file on the S3 bucket from any of the EC2 instances created in the previous exercise.
+
+# Use CLI to create a new S3 bucket
+
+- Create bucket "anwickes-bucket"
+> aws s3api create-bucket --bucket anwickes-devopsacademyau-bucket --region ap-southeast-2 --create-bucket-configuration LocationConstraint=ap-southeast-2
+
+```
+{
+    "Location": "http://anwickes-devopsacademyau-bucket.s3.amazonaws.com/"
+}
+```
+
+# Use CLI to upload file to S3 bucket created above
+
+- Upload file "index.html"
+> aws s3 cp index.html s3://anwickes-devopsacademyau-bucket
+
+```
+upload: .\index.html to s3://anwickes-devopsacademyau-bucket/index.html
+```
+
+> aws s3 ls s3://anwickes-devopsacademyau-bucket
+
+```
+2020-07-12 16:50:06        171 index.html
+```
+
+# Create IAM role that grants S3 access and attach to EC2 instance
+
+- Create role
+> aws iam create-role --role-name anwickes-devopsacademyau-bucket-role --assume-role-policy-document file://anwickes-devopsacademyau-bucket-role.json
+
+```
+{
+        "RoleId": "AROAYVE3JVV7MHKVLXFEW",
+        "Arn": "arn:aws:iam::595178663294:role/anwickes-devopsacademyau-bucket-role",
+        "CreateDate": "2020-07-12T07:07:47+00:00",
+        "AssumeRolePolicyDocument": {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Principal": {
+                        "Service": "ec2.amazonaws.com"
+                    },
+                    "Action": "sts:AssumeRole"
+                }
+            ]
+        }
+    }
+}
+```
+
+- Create policy and attach it to role
+> aws iam put-role-policy --role-name anwickes-devopsacademyau-bucket-role --policy-name anwickes-devopsacademyau-bucket-ro-policy --policy-document file://anwickes-devopsacademyau-bucket-ro-policy.json
```

Comment:
Same file, missing the JSON file for the policy document

Reply:
Added.